functional/ldap_test.go

```diff
@@ -123,3 +125,52 @@ func TestConnectorLDAPHealthy(t *testing.T) {
 		}
 	}
 }
+
+func TestLDAPPool(t *testing.T) {
+	server := ldapServer(t)
+	ldapPool := &connector.LDAPPool{
+		MaxIdleConn: 30,
```

Comment:
Would be cool if this was a benchmark, see how tuning this number effects the throughput. (For a follow up PR)